app.rb

```diff
@@ -0,0 +1,56 @@
+require 'sinatra/base'
+require 'pg'
+
+require_relative './spec/setup_test_database.rb'
+require_relative './lib/peep.rb'
+require_relative './lib/user.rb'
+
+# require_relative
+
+class Chitter < Sinatra::Base
```

Comment:
Clean and tidy controller. Good job of keeping app.rb slim and refactoring the code into the model files.